src/components/Passenger/Passenger.tsx

```diff
@@ -0,0 +1,44 @@
+import styles from "./Passenger.module.css";
+import { useState } from "react";
+import type { PassengerProps } from "./Passenger.definitions";
+
+/**
+ * Passenger component - a simple component that holds passenger details
+ * @property {string} firstName
+ * @property {string} lastName
+ * @property {string} email
+ * @property {string} relationship
+ * @property {string} userType
+ * @property {string} dateOfBirth
+ */
+const Passenger = (Passengers: PassengerProps) => {
+  const [number, setNumber] = useState<number>(0);
+
+  return (
+    <div className={styles.passengerCard}>
+      <div className={styles.title}>Passenger {}</div>
+      <div className={styles.name}>
+        {" "}
+        {Passengers.firstName} {Passengers.lastName}{" "}
+      </div>
+      <div className={styles.dob}>
+        <span className={styles.boldText}>DOB: </span>
+        {Passengers.dateOfBirth} {"(21)"}
```

Comment:
Let's bold the number and calculate the age from the date. You can assume the date is formatted as an `ISOString`